<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setter</class>
 <widget class="QDialog" name="Setter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>701</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>701</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>701</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MIB Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>640</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ACKTimeout">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_ACKTimeout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ACK Timeout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_EOFRecv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue EOF-Received Indication?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_NAKTimeout">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NAK Timeout</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NAKLimit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>470</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_resumed">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Resumed Indication?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_transFin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Transaction-Finished Indication?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Node ID</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_fileRecv">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fileRecv">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue File-Segment-Received Indication?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_EOFSent">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_suspended">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_EOFRecv">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_transFin">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_save">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_suspended">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Suspended Indication?</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_chunkSize">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>560</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>65200</number>
   </property>
   <property name="singleStep">
    <number>4</number>
   </property>
   <property name="value">
    <number>65200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_NAKLimit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NAK Limit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_EOFSent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue EOF-Sent Indication?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_inactivity">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inactivity Timeout</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ID">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_fileSent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue File-Segment-Sent Indication?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Incomplete Files?</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ACKLimit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_resumed">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_chunkSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>560</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Outgoing File Chunk Size (Bytes)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ACKLimit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ACK Limit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_NAKTimeout">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_inactivity">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>530</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>86400</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_fileSent">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Description</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_description">
      <property name="text">
       <string>The Management Information Base (MIB) contains such
information as default values for user communications
requirements (e.g., for address mapping, and for
communication timer settings).</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox_EOFSent</tabstop>
  <tabstop>comboBox_EOFRecv</tabstop>
  <tabstop>comboBox_fileSent</tabstop>
  <tabstop>comboBox_fileRecv</tabstop>
  <tabstop>comboBox_resumed</tabstop>
  <tabstop>comboBox_suspended</tabstop>
  <tabstop>comboBox_transFin</tabstop>
  <tabstop>comboBox_save</tabstop>
  <tabstop>spinBox_ACKLimit</tabstop>
  <tabstop>spinBox_ACKTimeout</tabstop>
  <tabstop>spinBox_NAKLimit</tabstop>
  <tabstop>spinBox_NAKTimeout</tabstop>
  <tabstop>spinBox_inactivity</tabstop>
  <tabstop>spinBox_chunkSize</tabstop>
  <tabstop>comboBox_ID</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Setter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Setter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
